compiler/parser/statement_parsing.go

```diff
@@ -117,6 +117,10 @@ func (p *Parser) parseParameters() []ast.Expression {
 		p.nextToken()
 		p.nextToken()
 		param := p.parseExpression(NORMAL)
+
+		if containsArg(params, param) {
```

Comment:
I recommend renaming it to `paramDuplicated` which will be more clear as a condition.